content/en/docs/setup/platform-setup/gke/index.md

```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
     the `gcloud container clusters create` command.
     {{< /warning >}}
 
+    {{< warning >}}
+    For private GKE Clusters, the firewall rule created does not open a port (15017) needed by validation webhook.
```

Comment:
lowercase clusters

maybe rephrase for easier translation: "the firewall rule that is created does not open the port (15017) needed by the validation webhook for master access."